css/index.css

```diff
@@ -68,9 +74,32 @@ p {
 
 .container {
     width: 800px;
+    max-width: 100%;
     margin: 0 auto;
 }
 
+.header {
+    width: 100%;
+    display: flex;
+    margin: 20px 0;
+}
+
+.header nav a {
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-end;
+    font-size: 2.0 rem;
+    text-decoration: none;
+    color: black;
+}
+
```

Comment:
So if we change the code slightly:
```
header {
    width: 100%;
    display: flex;
    margin: 20px 0;
}

header nav {
    width: 100%;
    display: flex;
    justify-content: space-around;
    align-items: flex-end;
    font-size: 2.0 rem;
    text-decoration: none;
    color: black;
}
```
This ends up looking a lot better. The element name is `header` not a class on the object. setting the width to `100%` on the a tag makes it take up the whole row.
![before changes](https://user-images.githubusercontent.com/12449668/37314092-7b9222d8-260f-11e8-80ea-94b2e22cc21f.png)
VS.
![after css changes](https://user-images.githubusercontent.com/12449668/37314084-6b01c1bc-260f-11e8-9aaa-4f74b314acea.png)